pennylane/hf/convert.py

```diff
@@ -0,0 +1,318 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for computing the molecular Hamiltonian.
+"""
+import numpy as np
+import openfermion
+import pennylane as qml
+from openfermion import QubitOperator
+from pennylane import Hamiltonian
+from pennylane.wires import Wires
+
+
+def _process_wires(wires, n_wires=None):
+    r"""Checks and consolidates custom user wire mapping into a consistent, direction-free, `Wires`
+    format. Used for converting between OpenFermion qubit numbering and Pennylane wire labels.
```

Comment:
```suggestion
    r"""Checks and consolidates custom user wire mapping into a consistent, direction-free, ``Wires``
    format. Used for converting between OpenFermion qubit numbering and PennyLane wire labels.
```

Reply:
Even though this doesn't show up in the docs, might still be good to be consistent with the double backticks! There are a couple more places below.